admin_guide/scoped_tokens.adoc

```diff
@@ -11,39 +11,43 @@
 toc::[]
 
 == Overview
-A user may wish to give another entity the power to act as him, but only in a limited way.
-For example, a project admin may wish to delegate the power to create pods.  One way to do 
-this is to create a scoped token.
-
-A scoped token, is a token that identifies as a given user, but is limited to certain actions 
-by its scope.  Right now, only a cluster-admin can create scoped tokens.
+A user may wish to give another entity the power to act as him or her, but only
+in a limited way. For example, a project administrator may want to delegate the
+power to create pods. One way to do  this is to create a scoped token.
 
+A scoped token is a token that identifies as a given user, but is limited to
+certain actions  by its scope.  ight now, only a *cluster-admin* can create
+scoped tokens.
 
+[[admin-guide-scoped-tokens-evaluation]]
 == Evaluation
-Scopes are evaluated by converting the set of scopes for a token into a set of PolicyRules.
-The request is then matched against those rules.  The request attributes must match at least 
-one of the scope rules to be passed to the "normal" authorizer for further authorization checks.
-
-
-== Scopes
-=== User Scopes
-User scopes are focused on getting information about a given user.  They are intent-based, so
-the rules are automatically created for you.
-
- 1. `user:info` - Allows access to information the user: name, groups, etc.
- 2. `user:check-access` - Allows access to self-localsubjectaccessreviews and self-subjectaccessreviews.
+Scopes are evaluated by converting the set of scopes for a token into a set of
+*PolicyRules*. Then, the request is matched against those rules. The request
+attributes must match at least  one of the scope rules to be passed to the
+"normal" authorizer for further authorization checks.
+
+[[admin-guide-scoped-tokens-user-scopes]]
+== User Scopes
+User scopes are focused on getting information about a given user. They are
+intent-based, so the rules are automatically created for you.
+
+* `user:info` - Allows access to information the user: name, groups, etc.
+* `user:check-access` - Allows access to *self-localsubjectaccessreviews* and *self-subjectaccessreviews*.
     Those are the variants where you pass an empty user and groups in your request object.
```

Comment:
should "variants" be "variables"? I don't know, I'm just asking. I don't think I remember seeing "variants" in our docs aside from choosing which version of OpenShift to install.